.github/workflows/UnitTestArm.yml

```diff
@@ -0,0 +1,38 @@
+name: Unit test for Arm
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  workflow_dispatch:
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        julia-version: ['1.0', '1', 'nightly']
+        os: [ubuntu-20.04]
+        distro: [ubuntu20.04]
+        arch: [aarch64]
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: uraimo/run-on-arch-action@v2.0.7
+        with:
+          arch: ${{ matrix.arch }}
+          distro: ${{ matrix.distro }}
+          githubToken: ${{ github.token }}
```

Comment:
is this for code coverage upload?

Reply:
This is used for pushing the cache images. However, I am not sure about its necessity.

Reply:
I was expecting python3 to be included in the docker images, but it appears not to be. Jill.py is a great tool if we already have python installed, but it has too large a footprint to be used just for installing Julia. :sweat_smile:
However, the original jill.sh does not support arm at the moment.

Reply:
> Jill.py is a great tool if we already have python installed, but it has too large a footprint to be used just for installing Julia.

Indeed. Unfortunately, I'm not a bash wizard to add the functionality I wanted (local release registry), so I have to use Python to do it.

I use this to set up CI in the self-hosted gitlab in my school servers, and with properly set up apt/jill/pip mirrors, I can make sure most network traffics are limited in the LAN network.